packages/video_player/video_player_windows/README.md

```diff
@@ -0,0 +1,15 @@
+# video\_player\_windows
+
+The Windows implementation of [`video_player`][1].
+
+## Usage
+
+This package is [endorsed][2], which means you can simply use `video_player`
```

Comment:
Do you think this is ready enough for prime-time that we'll want to endorse it now, or still more alpha (in which case we could hold off on endorsing it, and ask people to testing it out by depending on it directly). We don't have a bright line on this, it's mostly about whether it's basically feature complete.

Reply:
Do we have a list of features that would define a minimal viable product for endorsement? I'm pretty sure this is going to support everything that ExoPlayer supports for Android and iOS. There are 3 things on my mental todo list that still need to be "fixed", and/or at desired features:
1) V-Sync (still seeing some artifacts when screen is large enough), needs to be fixed before merging this.
2) m3u8 files. I know that Media Foundation supports this, but it is failing on the existing code path, so I removed the test that was validating it.
3) DRM content (not sure if this is a must, but should be possible at least when Composition is enabled and available, AFAIK)

My C++ is not the best, so I know this will have some level of mistakes/bugs, but once the PR is reviewed properly merged, I don't see it being less feature complete than the Android or iOS implementations.

Reply:
> Do we have a list of features that would define a minimal viable product for endorsement? I'm pretty sure this is going to support everything that ExoPlayer supports for Android and iOS.

Sounds feature-complete for endorsement to me :)

> 2. m3u8 files. I know that Media Foundation supports this, but it is failing on the existing code path, so I removed the test that was validating it.

This is definitely something we should track in an issue (I'm sure people will hit it), but I would not block endorsement on it.

> 3. DRM content (not sure if this is a must, but should be possible at least when Composition is enabled and available, AFAIK)

This doesn't work on any platform currently, since we are using textures rather than platform views, and DRM systems generally don't want to write frames into arbitrary buffers. (Eventually we will likely implement a platform-view-based alternative for DRM playback, but that's not something that would block this; based on various discussions our current expectation is that long term every platform will need both texture-based and platform-view-based playback options due to the tradeoffs inherent in platform views.)